src/main/java/com/eerussianguy/blazemap/feature/Overlays.java

```diff
@@ -26,21 +24,19 @@ public static void onRegisterOverlays(RegisterGuiOverlaysEvent event) {
     public static void renderMinimap(ForgeGui gui, GuiGraphics graphics, float partialTicks, int width, int height) {
         if (BlazeMapConfig.CLIENT.minimap.enabled.get()) {
             PoseStack stack = graphics.pose();
+
             stack.pushPose();
-            var buffers = MultiBufferSource.immediate(Tesselator.getInstance().getBuilder());
-            MinimapRenderer.INSTANCE.draw(graphics, buffers, gui, width, height);
-            buffers.endBatch();
```

Comment:
I wasn't quite sure what these `buffers` blocks were doing, and there wasn't a simple way to transfer them to the new system without understanding what they were. 

Given it doesn't seem to have broken anything, I assume it's ok? But otherwise if you can explain to me what this is supposed to do (googling didn't turn up anything useful), I can fix it up to be as before.